exporter/jaeger/jaeger.go

```diff
@@ -39,6 +39,10 @@ type Options struct {
 	// For example, http://localhost:14268.
 	Endpoint string
 
+	// LocalAgentHostPort instructs exporter to send spans to jaeger-agent at this address.
+	// For example, localhost:6831.
+	LocalAgentHostPort string
```

Comment:
Can we call this AgentEndpoint? As far as, I can understand, agent can work on UDP and TCP. Don't we need to give options to the user? 

Maybe it might make sense to have a net.Conn field.

// ...
AgentConn *net.Conn

Then, user can dial it themselves:

conn, err := net.Dial("tcp", "endpoint:port")

and we can type cast conn to see whether we should use the UDP or TCP endpoint.

Reply:
@rakyll Updated, As far as I know, agent only work on UDP. http://jaeger.readthedocs.io/en/latest/deployment/